pinecone/__init__.py

```diff
@@ -3,22 +3,10 @@
 """
 from .config import *
 from .exceptions import *
-from .manage import *
-from .index import *
+from .control.pinecone import Pinecone
+from .data.index import *
 
 try:
```

Comment:
These `try` import statements led to some pretty confusing debug situations when stuff started failing due to me moving packages from one location to another since the intention is really only to ignore failures caused by people installing without grpc extras. We should revisit this pattern and make sure we're testing stuff both with and without grpc deps loaded.